File: DataCenterWebApp/ViewModels/Instrument/InstrumentIdVM.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace iCDataCenterClientHost.ViewModels.Instrument
{
    [JsonObject(MemberSerialization.OptOut)]
    public class InstrumentIdVM : IInstrumentIdVM
    {
        #region Constructor
        public InstrumentIdVM()
        {

        }
        #endregion

        #region Properties
        public string vm_address { get; set; }
        public string vm_description { get; set; }
        #endregion
    }
}

File: DataCenterWebApp/ClientApp/src/app/Instruments/InstrumentIdVM.ts
import { IInstrumentIdVM } from '../Instruments/IInstrumentIdVM';

export class InstrumentIdVM implements IInstrumentIdVM {
  vm_address: string;
  vm_description: string;

  constructor(address: string, description: string) {
    this.vm_address = address;
    this.vm_description = description;
  }
}
